docs/scaling.md

```diff
@@ -0,0 +1,265 @@
+2. Then we must take all the resultant virtual datasets (one per file), and combine them together into one final virtual dataset. This is the reduce step.
+
+Finally we write this single virtual dataset to some persistent format.
+We have already reduced the data, so this step is a third step, the serialization step.
+
+In our case the amount of data being reduced is fairly small - each virtual dataset is hopefully only a few kBs in memory, small enough to send over the network.
+Even a million such virtual datasets together would only require a few GB of RAM in total to hold in memory at once.
+This means that as long as we can get all the virtual datasets to be sent back successfully, the reduce step can generally be performed in memory on a single small machine, such as a laptop.
+This avoids the need for more complicated parallelization strategies such as a tree-reduce.
+
+## Parallelization Approaches
+
+There are two ways you can implement a map-reduce approach to virtualization in your code.
+The first is to write it yourself, and the second is to use `open_virtual_mfdataset`.
+
+### Manual parallelism
+
+You are free to call `open_virtual_dataset` on your various files however you like, using any method to apply them, including applying them in parallel.
+
+For example you may want to parallelize using the [dask library](https://www.dask.org/), which you can do by wrapping each call using `dask.delayed` like this:
+
+```python
+import virtualizarr as vz
+import dask
+
+tasks = [dask.delayed(vz.open_virtual_dataset)(path) for path in filepaths]
+virtual_datasets = dask.compute(tasks)
+```
+
+This returns a list of virtual `xr.Dataset` objects, which you can then combine:
+
+```python
+import xarray as xr
+
+combined_vds = xr.combine_by_coords(virtual_datasets)
+```
+
+### The `parallel` kwarg to `open_virtual_mfdataset`
+
+Alternatively, you can use `virtualizarr.open_virtual_mfdataset`'s `parallel` keyword argument.
+
+This argument allows you to conveniently choose from a range of pre-defined parallel execution frameworks, or even pass your own executor.
+
+The resulting code only takes one function call to generate virtual references in parallel and combine them into one virtual dataset.
+
+```python
+combined_vds = vz.open_virtual_mfdataset(filepaths, parallel=<choice_of_executor>)
+```
+
+VirtualiZarr's `open_virtual_mfdataset` is designed to mimic the API of Xarray's `open_mfdataset`, and so accepts all the same keyword argument options for combining.
+
+## Executors
+
+VirtualiZarr comes with a small selection of executors you can choose from when using `open_virtual_mfdataset`.
+
+!!!note
+If you prefer to do manual parallelism but would like to use one of these executors you can - just import the executor directly from the `virtualizarr.parallel` namespace and use its `.map` method.
+```
```

Comment:
```suggestion
    If you prefer to do manual parallelism but would like to use one of these executors you can - just import the executor directly from the `virtualizarr.parallel` namespace and use its `.map` method.
```
mkdocs admonitions use intend rather than backticks